nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestFetchFile.java

```diff
@@ -48,6 +48,19 @@ public void prepDestDirectory() throws IOException {
     }
 
     @Test
+    public void notFound() throws IOException {
+        final File sourceFile = new File("notFound");
```

Comment:
That is correct. We can only test it in some specially set integration environment where you may have a remote share that is known to the OS but temporarily unreachable. That is where the new code becomes relevant.